generate_from_csv.py

```diff
@@ -163,7 +171,18 @@ def generate_from_csv(args):
                 warnings.warn("{}: {} travelling in time (departing {})".format(
                         arrival, v_name, departure))
 
-            events["vehicle_events"].append({
+            # adjust SoC if delta_soc < min_soc
+            if args.min_soc < delta_soc:
```

Comment:
This does not work. In my understanding delta_soc is calculated for every single trip. But since there is not always a charging station available at the end of a trip (defined by the column "connect_cs" in the input-csv) another argument has to be used here.
args.min_soc should be compared with the sum of all delta_soc up to the moment when a charging station is available. As it is done in the strategy-simulation later.